import java.util.*;
class ExceptionProg1{
	public static void main(String[] args){
		System.out.println("Enter the number of Elements in the array:");
		Scanner sc=new Scanner(System.in);
		try{
			int n=sc.nextInt();
			int [] arr=new int[n];
			System.out.println("Enter the elements in the array:");
			for(int i=0;i<n;i++)
				arr[i]=sc.nextInt();
			System.out.println("Enter the index of the array ");
			int index=sc.nextInt();
			System.out.println("The array element at index "+index+"="+arr[index]);
			System.out.println("The array element successfully accessed");	
		}catch (ArrayIndexOutOfBoundsException aoobe){
			System.out.println("java.lang.ArrayIndexOutOfBoundsException");
		}
		catch (InputMismatchException ime){
			System.out.println("java.util.InputMismatchException");
		}
		catch (NumberFormatException nfe){
			System.out.println("java.util.NumberFormatException");
		}
	}
}
